### PR TITLE
store: add some function for store state to decrease the same code.

### LINKED ISSUE #4901: remove some repeated code about store state.
## Enhancement Task
add two function to filter up store and tiflash

### PR DESCRIPTION
**Pull request by bufferflies**
### What problem does this PR solve?

Issue Number: Close #4901

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Unit test

Code changes

Side effects

Related changes

### Release note

```release-note
 None.
```

### CODE REVIEW

**Review by AndreMouche — COMMENTED** [review]
LGTM good job

**Review by AndreMouche — APPROVED** [review]
LGTM

**Review by lhy1024 — APPROVED** [review]

File: pkg/autoscaling/calculation.go [hunk 1]
```diff
@@ -136,7 +136,7 @@ func filterTiKVInstances(informer core.StoreSetInformer) []instance {
 	var instances []instance
 	stores := informer.GetStores()
 	for _, store := range stores {
-		if store.IsPreparing() || store.IsServing() {
+		if store.IsUp() {
```
Comment (lhy1024) [inline pkg/autoscaling/calculation.go:139]: cc @rleungx

**Pull request merged**